src/deepsparse/benchmark/benchmark_pipeline.py

```diff
@@ -346,6 +346,11 @@ def _clear_measurements():
             "Generated no batch timings, try extending benchmark time with '--time'"
         )
 
+    if SupportedTasks.is_text_generation(task) or SupportedTasks.is_code_generation(
+        task
+    ):
+        kwargs.pop("middleware_manager")
```

Comment:
Will middleware be only present for text-gen/code-gen pipelines?

Reply:
Yes. We haven't migrated any of the other pipelines to use v2. Only these two tasks.